src/Employer/Employer.Web/Controllers/VacancyPreviewController.cs

```diff
@@ -119,7 +118,9 @@ private int GetSectionStateCount(VacancyPreviewViewModel viewModel)
 
         private bool CheckIfSectionIsIncomplete(VacancyPreviewSectionState viewModelTitleSectionState)
         {
-            return viewModelTitleSectionState == VacancyPreviewSectionState.Incomplete;
+            if (viewModelTitleSectionState == VacancyPreviewSectionState.Incomplete)
+                return viewModelTitleSectionState == VacancyPreviewSectionState.Incomplete;
+            return viewModelTitleSectionState == VacancyPreviewSectionState.InvalidIncomplete;
```

Comment:
This is a bit odd to read. Isn't this just an `or`?
`return viewModelTitleSectionState == VacancyPreviewSectionState.InvalidIncomplete || viewModelTitleSectionState == VacancyPreviewSectionState.Incomplete`